lib/models/task.js

```diff
@@ -6,6 +6,17 @@ class Task extends CoreObject {
   run(/*options*/) {
     throw new Error('Task needs to have run() defined.');
   }
+
+  /**
+   * Interrupt comamd with an exit code
+   * Called when the process is interrupted from outside, e.g. CTRL+C or `process.kill()`
+   *
+   * @private
+   * @method onInterrupt
+   */
+  onInterrupt() {
+    process.exit(1);
```

Comment:
We still want to communicate exit status.  Letting handlers just call `process.exit` seems like the most straightforward way of doing that, although I agree this is mega troll for lib usage.

Having said that, since we only have two statuses at the moment we could determine status from the final state of the command promise.  Otherwise seems we need some other out of band way to communicate exit status.